site/content/blog/imgpkg-tags.md

```diff
@@ -0,0 +1,219 @@
+
+In this output you can see that `imgpkg` lists all the files that will be pushed to the Registry as well has the Fully
+Qualified Image Reference for the OCI Image created.
+
+One thing that you might notice is that despite the fact that this Bundle points to another OCI
+Image, [check the ImagesLock file](https://github.com/vmware-tanzu/carvel-imgpkg/blob/develop/examples/basic-step-2/.imgpkg/images.yml)
+, nothing happens to it.
+
+But why? As we previously stated, pushing a Bundle to a Registry only creates the OCI Image that defines the Bundle.
+
+## Copying a Bundle to a different Registry
+
+Having a Bundle in a particular Registry might not be good enough, and we might need to Promote our software to
+a different Registry, or even copy this Bundle to a public Registry so that other people can use the software being
+developed.
+
+One of the most awesome features of the `imgpkg` Bundles is that despite the fact that Bundles are just OCI Images that
+contain some metadata they also can aggregate other OCI Images. Let us look at the prior Bundle, when we check the
+ImagesLock file we can see that it contains a reference to a different OCI
+Image `index.docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0`
+
+```shell
+$ cat examples/basic-step-2/.imgpkg/images.yml
+apiVersion: imgpkg.carvel.dev/v1alpha1
+kind: ImagesLock
+images:
+- image: index.docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+  annotations:
+    kbld.carvel.dev/id: docker.io/dkalinin/k8s-simple-app
+```
+
+So whenever we are copying a Bundle, `imgpkg` does the leg work and copies all the Images or Bundles that are referenced
+by the Bundle being copied. This way with a single command you can copy all the OCI Images associated with the Bundle to
+the destination. Let us see `imgpkg` in action
+
+```shell
+$ imgpkg copy -b localhost:5000/cool-new-bundle:my-tag --to-repo my-company.repo.io:5000/destination-repo
+copy | exporting 2 images...
+copy | will export index.docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+copy | will export localhost:5000/cool-new-bundle@sha256:5c2dafe3c70c13990190d643c91e9f67b8129b179257674888178868474f6511
+copy | exported 2 images
+copy | importing 2 images...
+
+ 
+copy | done uploading images
+
+Succeeded
+```
+
+From the output we can see that 2 OCI Images were copied, the Bundle Image and the Image that is in the ImagesLock.
+
+When we copied we only provided one destination Repository and `imgpkg` will copy all the OCI Images into that
+repository. This was a design decision that was made due to the following constraints:
+
+- Multiple images that have the same Image Name but are not the same. Assuming we have 2 bundles one that contains the
+  image my.registry.io/controller@sha256:aaaa and another that contains the image other.registry.io/controller@sha256:
```

Comment:
For consistency, I would consider putting the image references in `code` format.